src/Main.java

```diff
@@ -1,7 +1,48 @@
 public class Main {
     public static void main(String[] args) {
 
-        // Тут має розгортатися епічна битва між Супергероями
+        SuperHero ironman = new SuperHero();
+        SuperHero flash = new SuperHero();
 
+        System.out.println("Епічна битва супергероїв розпочинається!");
+// В мене Марвел рулить))
+        ironman.name = "IronMan";
+        ironman.attackPower = 35;
+        ironman.defensePower = 30;
+
+        flash.name = "Flash";
+        flash.attackPower = 25;
+        flash.defensePower = 20;
+
+        // Lets get it started))
+
+        int round = 1;
+        //System.out.println("Раунд " + round);
+
+        while (flash.isAlive() && ironman.isAlive()) {
+            System.out.println("Round " + round);
+            flash.attack(ironman);
+            if (!ironman.isAlive()) break;
+            ironman.attack(flash);
+            round++;
+        }
+
+
+        //ironman.attack(flash); // це я собі так писала після уроку для особистого розуміння
+        //flash.attack(ironman);
+        //flash.attack(ironman);
+        //ironman.attack(flash);
+        //ironman.attack(flash);
+
+        if (flash.health < ironman.health){
```

Comment:
wait a minute... we have a fight for life, until only one remains alive

So here it would be better to check not who has more health, but rather who's left alive :)

Reply:
In addition, your `while` loop that starts on line 22 can have only one chance to break - if somebody dies -> so at line 37 you will have at least one dead one alive